utils.py

```diff
@@ -650,6 +650,21 @@ def unescape_encoded_uri_component(escaped_string):
     return python_utils.urllib_unquote(escaped_string).decode('utf-8')
 
 
+def snake_case_to_camel_case(snake_str):
+    """Converts a string in snake_case to camelCase.
+
+        Args:
```

Comment:
The indentation here is wrong; follow other docstrings in the rest of the Python code.

Reply:
Done